target/linux/ath79/image/generic.mk

```diff
@@ -40,6 +40,10 @@ define Build/append-md5sum-bin
 		xargs echo -ne >> $@
 endef
 
+define Build/append-string
+       echo -n $(1) >> $@
+endef
+
```

Comment:
That's redundant, is this is also present in include/image-commands.mk.
Look at rut955 in this file, it's already using it.